src/app/dashboard/_dashboard.scss

```diff
@@ -7,35 +8,114 @@ $paperCardBackgroundColor: #212121 !default;
 
 #dashboardContainer {
 	background-color: $dashboardContainerBackgroundColor;
+	height: 100%;
 
-	.userbar {
+	// typographic defaults
+	font-family: Roboto, sans-serif;
+	font-style: normal;
+	font-size: 1em;
+	line-height: 1.2em;
+
+	header {
 		position: fixed;
 		top: 0;
-		left: 0;
+		display: block;
 		width: 100%;
-		height: 46px;
 		background-color: $userbarBackgroundColor;
-		padding: 5px;
-		margin-bottom: 10px;
+		text-color: $userbarTextColor;
+		box-shadow: rgba(0, 0, 0, 0.25) 0px 2px 5px 0px;
+		margin: 0;
+
+			nav {
+			top: 0;
+			left: 0;
+			padding: 0;
+
+				ul {
+				list-style-type: none;
+				display: block;
+				float: right;
+				margin: 0;
+				padding: 0;
+
+				li {
+					display: block;
+					position: relative;
+					
+					a {
+						display: block;
```

Comment:
make sure you actually need to put `display: block;` on these elements.  i see you repeating that line multiple times in nested content